contracts/BC_fusion/System.sol

```diff
@@ -3,6 +3,9 @@ pragma solidity 0.8.17;
 
 contract System {
     /*----------------- constants -----------------*/
+    uint8 public constant STAKING_CHANNELID = 0x08;
+    uint8 public constant BC_FUSION_CHANNELID = 0x11;
```

Comment:
Please comment what is the usage of this channel, as `BC_FUSION_CHANNEL` can not explain it.

Reply:
`new channel id for cross-chain stake migration after BC-fusion` ==> `new channel id for cross-chain redelegate from Beacon Chain to Smart Chain after Feyman upgrade`